docs/trex_getdata.md

```diff
@@ -72,15 +73,53 @@ After you have a worksheet object, you can call one of the methods to access the
 
 ## Get summary data from a worksheet
 
+Starting with Tableau 2022.4 and the Dashboard Extensions API library v1.10, you should use the `getSummaryDataReaderAsync()` method to get data from a worksheet. This method returns a `DataTableReader` object that you can use to access the data.
+
+If your summary data is less than 4,000,000 rows (400 pages), you can use the `DataTableReader.getAllPagesAsync()` method to retrieve a single `DataTable` from the `DataTableReader`. The following example shows how to do this.
+
+```javascript
+
+// Use `await` only inside an `async` method
+
+let vizActiveSheet = viz.workbook.activeSheet;
+if (vizActiveSheet.sheetType === "worksheet") {
+  const dataTableReader = await vizActiveSheet.getSummaryDataReaderAsync();
```

Comment:
Remove 84 and 85
Change 86 to be await worksheet.getSummaryDataReaderAsync()

Reply:
done